src/App.js

```diff
@@ -36,6 +34,12 @@ function App() {
   const [mentors, setMentors] = useState(mentorsList);
   const [darkMode, setDarkMode] = useState(false);
 
+  const appliedTheme = createMuiTheme({
```

Comment:
I would like to change the basic theme too. I am gonna open an issue for that and would like to see some suggestions on the color palettes. I 'd like to assign this to you.